sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -227,21 +227,40 @@ with open("<path to your receipt>", "rb") as fd:
 poller = form_recognizer_client.begin_recognize_receipts(receipt)
 result = poller.result()
 
-r = result[0]
-print("Receipt contained the following values with confidences: ")
-print("Receipt Type: {} has confidence: {}".format(r.receipt_type.type, r.receipt_type.confidence))
-print("Merchant Name: {} has confidence: {}".format(r.merchant_name.value, r.merchant_name.confidence))
-print("Transaction Date: {} has confidence: {}".format(r.transaction_date.value, r.transaction_date.confidence))
+receipt = result[0]
+print("Receipt Type: {} has confidence: {}".format(receipt.receipt_type.type, receipt.receipt_type.confidence))
```

Comment:
should `receipt_type` be strongly typed?

edit: asking Anne for clarity on this

Reply:
After guidance from @annelo-msft, we will treat this as any other field